kubeflow/hack/delete_gcp.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
-set -ex
+set -e
 
 if [[ -z "${MGMTCTXT}" ]]; then
     echo "MGMTCTXT env var is required"
     exit 1
 fi
 
-echo "Deleting all GCP resources will cause destruction of all services and data on this cluster. Confirm? [y/N]";
+echo "Deleting all Google Cloud resources including your GKE cluster and data in the cluster, except the Cloud SQL instance and GCS bucket. Confirm? [y/N]";
 read REPLY;
```

Comment:
I think we can merge these two lines by:

read -p "Deleting all ..." REPLY;

I will make this change from my side.